launch/.gitignore

```diff
@@ -0,0 +1,3 @@
+.yarn/
+.yarnrc.yml
```

Comment:
`.yarnrc.yml` actually need to be kept in git, you can refer to stake-client & parallel-dapp. So that we are sure no surprises